### PR TITLE
[FEATURE] Hide bookmarks for v1 launch

### PR DESCRIPTION
**Pull request by ohwhen**
This PR introduces a FeatureFlags to be used to enable/disable features within the app. Additionally it uses the FeatureFlags.boomarksEnabled in two places to hide the button to bookmark a person.

Screen Shot 2020-06-07 at 3 34 47 PM

### CODE REVIEW

File: SayTheirNames/Source/FeatureFlags.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import Foundation
+
+struct FeatureFlags {
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/FeatureFlags.swift:27]: What if we made this a dependency instead so they could be configured by the backend in the future?

Comment (hybridcattt) [reply SayTheirNames/Source/FeatureFlags.swift:27]: Love this approach, and yes!
That's exactly how I usually architect apps too. Though here figured to do quick and dirty, so will close my #117

Comment (hybridcattt) [reply SayTheirNames/Source/FeatureFlags.swift:27]: Not hiding filters yet? Are you planning to do it next?

Comment (ohwhen) [reply SayTheirNames/Source/FeatureFlags.swift:27]: Yeah good idea, let's discuss it with the backend team 👍  

To keep things moving, let's land this and we can quickly change it once the API is done?

Comment (hybridcattt) [reply SayTheirNames/Source/FeatureFlags.swift:27]: of course! actually what I suggested was regardless of the API - flags can be local but still be organised in the dependency container to keep things in one place.

**Pull request merged**